dotcom-rendering/.storybook/main.ts

```diff
@@ -21,6 +21,7 @@ const config: StorybookConfig = {
 	staticDirs: ['../src/static'],
 	addons: [
 		'@storybook/addon-essentials',
+		'@storybook/addon-interactions',
```

Comment:
Adds an interactions tab to the Storybook UI https://storybook.js.org/f3b4ad3cac0565f5aa2c17b22317091b/addon-interaction-example-optimized.mp4